src/minit-assets.php

```diff
@@ -248,12 +248,6 @@ protected function get_asset_relative_path( $handle ) {
 			if ( ! empty( $src_parts ) ) {
 				return array_pop( $src_parts );
 			}
-		} elseif ( ! empty( $this->handler->registered[ $handle ]->extra[ 'path' ] ) ) {
```

Comment:
Assets with path but _without_ URL are specific things (such as individual block styles) that we shouldn't consider for the bundle. In particular, we don't want to include page-specific block styles as they would bust the bundle and make it unusable between requests.